common-util/useFetchApi.js

```diff
@@ -13,12 +13,23 @@ const useFetch = (subUrl, params) => {
   return { data, isLoading };
 };
 
-export const useFetchVideos = (limit = 1000) => {
-  const params = {
-    sort: ['date:desc'],
-    populate: '*',
-    'pagination[limit]': limit,
-  };
+export const useFetchVideos = (limit = 1000, isPodcast = false) => {
+  const params = isPodcast
+    ? {
+        sort: ['date:desc'],
+        populate: '*',
+        'pagination[limit]': limit,
+        filters: {
+          title: {
+            $contains: 'Ep.',
+          },
+        },
+      }
+    : {
+        sort: ['date:desc'],
+        populate: '*',
+        'pagination[limit]': limit,
+      };
```

Comment:
tiny and up to you, could be just below to safe some space + for readability
```suggestion
  const params = {
        sort: ['date:desc'],
        populate: '*',
        'pagination[limit]': limit,
        filters: isPodcast ? {
          title: {
            $contains: 'Ep.',
          },
        } : undefined,
      }
```